docs/config/README.md

```diff
@@ -1225,6 +1225,7 @@ The `direnv` module shows the status of the current rc file if one is present. T
 | `detect_files`      | `['.envrc']`                           | Which filenames should trigger this module.           |
 | `detect_folders`    | `[]`                                   | Which folders should trigger this module.             |
 | `allowed_msg`       | `'allowed'`                            | The message displayed when an rc file is allowed.     |
+| `not_allowed_msg`   | `'not_allowed'`                        | The message displayed when an rc file is not_allowed. |
```

Comment:
The config has `not allowed`. Which value is it supposed to be?

Reply:
Ah, good catch. Should be “not allowed”

Reply:
```suggestion
| `not_allowed_msg`   | `'not allowed'`                        | The message displayed when an rc file is not_allowed. |
```